### PR TITLE
What's On: Exhibition page

### PR DESCRIPTION
**Pull request by jamesbiggs**
## About these changes

This PR adds the Exhibition page to the What's On project.

See the page spec here: https://national-archives.atlassian.net/wiki/spaces/DSWhatson/pages/156368897/Exhibition+page+specification

## How to check these changes

Go into the What's On page in the CMS, and add a child page. It will give you the option of an Event page or an Exhibition page. Pick an Exhibition page and you will then be able to fill in the fields which are spec'd above.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [x] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: etna/whatson/models.py [hunk 1]
```diff
@@ -829,3 +834,314 @@ class EventFilterForm(forms.Form):
             attrs={"class": "filters__toggle-input", "data-js-family": ""}
         ),
     )
+
+
+class ExhibitionHighlights(Orderable):
```
Comment (jams2) [inline etna/whatson/models.py:839]: Question: should this be named `ExhibitionHighlight` (singular)? It looks like one instance of this class maps to a single "highlight".

File: etna/whatson/models.py [hunk 2]
```diff
@@ -829,3 +834,314 @@ class EventFilterForm(forms.Form):
...
+    LIGHT = "light", _("Light")
+    DARK = "dark", _("Dark")
+
+
+class ExhibitionPage(ArticleTagMixin, TopicalPageMixin, BasePageWithIntro):
+    """ExhibitionPage
+
+    An event where editors can create exhibitions. Exhibitions do not come
+    from Eventbrite - they are internal TNA exhibitions.
+    """
```
Comment (jams2) [inline etna/whatson/models.py:891]: Good context :+1:

File: etna/whatson/models.py [hunk 3]
```diff
@@ -829,3 +834,314 @@ class EventFilterForm(forms.Form):
...
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    # Related content
+    articles_title = models.CharField(
+        max_length=100,
+        blank=True,
+        help_text=_("The title to display for the articles section."),
+    )
```
Comment (jams2) [inline etna/whatson/models.py:1006]: Question: is there a need to limit this to 100 characters? That's verging on the side of restrictively short.

File: etna/whatson/blocks.py [hunk 4]
```diff
@@ -1,7 +1,23 @@
...
 
-from etna.core.blocks import ParagraphBlock
+from etna.core.blocks import PageListBlock, ParagraphBlock
 
 
 class EventPageBlock(blocks.StreamBlock):
     paragraph = ParagraphBlock()
+
+
+class RelatedArticlesBlock(blocks.StreamBlock):
```
Comment (jams2) [inline etna/whatson/blocks.py:10]: Question: is this for use on ExhibitionPages only? If so it might be helpful to name it more specifically.

**Review by jams2 — COMMENTED** [review]
Looking good @jamesbiggs , just a few comments.

File: etna/whatson/blocks.py (hunk 4 above)
Comment (jamesbiggs) [reply etna/whatson/blocks.py:10]: I think at some point we might add it to other pages - we tend to make things specific and then expand them to be generic, which makes it a bit more difficult to change down the line, so we've been trying to go a bit more "generic" with naming recently 😄 Good point though, I agree!

**Review by jams2 — APPROVED** [review]

**Pull request merged**